packages/frontend-plugin-api/api-report.md

```diff
@@ -18,7 +18,23 @@ import { ZodTypeDef } from 'zod';
 
 // @public (undocumented)
 export type AnyExtensionDataMap = {
-  [name in string]: ExtensionDataRef<any, any>;
+  [name in string]: ExtensionDataRef<
+    unknown,
+    {
+      optional?: true;
```

Comment:
perhaps a bit funny how this one ended up being solved differently than the one below

Reply:
Yep I actually tried to refactor this one, but realized we specifically designed it like this so that the API reports for `ExtensionDataRef`s would be less verbose. Not really an issue for inputs, and the below solution is a bit more straightforward imo